pyi.py

```diff
@@ -247,6 +247,15 @@ def _check_platform_check(self, node: ast.Compare) -> None:
         else:
             self.error(node, Y007)
 
+    def visit_ClassDef(self, node: ast.ClassDef) -> None:
+        self.generic_visit(node)
+
+        for i, statement in enumerate(node.body):
```

Comment:
We can make this check slightly stronger: `pass` should never appear in a class body, and if there is more than one statement, none of them should be `...`.

Reply:
That sounds a good idea!

Reply:
Updated based on your suggestions!